ambry-api/src/main/java/com/github/ambry/rest/RestUtils.java

```diff
@@ -201,7 +201,10 @@
      * stitched together.
      */
     public static final String CHUNK_UPLOAD = "x-ambry-chunk-upload";
-
+    /**
+     *Boolean field set to "true" to indicate that this is an upload of a data chunk of a stitched upload(for named blob only).
```

Comment:
I think this comment should be changed to indicate that it is set to a string that indicates STITCH request vs regular upload

Reply:
I think the name should also be changed to be more specific.  I'd never have guessed what it apparently means.

Reply:
Changed the name to UPLOAD_NAMED_BLOB_MODE and updated the comments as well.